migrate/migrate.go

```diff
@@ -148,12 +152,44 @@ func runScripts(pool *sql.DB, scripts map[string]string, ignoreFiles []string) e
 		filenames = append(filenames, name)
 	}
 	sort.Strings(filenames)
+
 	for _, file := range filenames {
+		content, ok := scripts[file]
+		if !ok {
+			continue
+		}
```

Comment:
Should we not raise error here ?

Reply:
`filenames` are generated from scripts so I don't think we will ever have `ok==false`